cocos/platform/ios/CCEAGLView-ios.mm

```diff
@@ -420,17 +420,27 @@ - (void)touchesMoved:(NSSet *)touches withEvent:(UIEvent *)event
     UITouch* ids[IOS_MAX_TOUCHES_COUNT] = {0};
     float xs[IOS_MAX_TOUCHES_COUNT] = {0.0f};
     float ys[IOS_MAX_TOUCHES_COUNT] = {0.0f};
+    float fs[IOS_MAX_TOUCHES_COUNT] = {0.0f};
+    float ms[IOS_MAX_TOUCHES_COUNT] = {0.0f};
     
     int i = 0;
     for (UITouch *touch in touches) {
         ids[i] = touch;
         xs[i] = [touch locationInView: [touch view]].x * self.contentScaleFactor;;
         ys[i] = [touch locationInView: [touch view]].y * self.contentScaleFactor;;
+
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_9_0 // User 9.0 or higher SDK compiled
```

Comment:
@ricardoquesada Yes, we need it. Because 3d touch is introduced in iOS SDK version 9.0, touch.force define as "@property(nonatomic,readonly) CGFloat force NS_AVAILABLE_IOS(9_0);" in UITouch.h, same as touch.maxinumPossibleForce, if it compiled with SDK under version 9.0, will get a compile error.